Amazon.QLDB.Driver/session/QldbSession.cs

```diff
@@ -107,14 +76,24 @@ internal T Execute<T>(Func<TransactionExecutor, T> func)
                 transaction.Commit();
                 return returnedValue;
             }
+            catch (TransactionAbortedException)
```

Comment:
This exception mapping logic seems to be duplicated in the Async session - can you deduplicate there as well?

Reply:
Having to await the abort call makes this not very feasible.